pvanalytics/features/shading.py

```diff
@@ -321,6 +321,9 @@ def fixed(ghi, daytime, clearsky, interval=None, min_gradient=2):
     interval : int, optional
         Interval between data points in minutes. If not specified the
         interval is inferred from the frequency of the index of `ghi`.
+    min_gradient : int, default 2
+        Threshold value for the morphological gradient.
+        TODO: what is a threshold value for a morphological gradient?
```

Comment:
Someone please suggest a description for this parameter, ideally mentioning why one might want to change its value from the default.  Also is "int" actually the appropriate type?

Reply:
I don't know that we can define that with any more clarity. Type should be float.

https://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.ndimage.morphology.morphological_gradient.html